data/json/items/chemicals_and_resources.json

```diff
@@ -1745,5 +1745,26 @@
     "price": 100,
     "price_postapoc": 5,
     "count": 100
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "liquid_soap",
+    "name": { "str_sp": "liquid soap" },
+    "weight": "260 g",
+    "color": "light_cyan",
+    "container": "bottle_bathroom",
+    "sealed": false,
+    "comestible_type": "DRINK",
+    "symbol": "~",
+    "quench": -90,
+    "healthy": -20,
+    "description": "Liquid soap.",
+    "price": 0,
+    "price_postapoc": 0,
+    "volume": "250 ml",
+    "phase": "liquid",
+    "category": "chems",
+    "freezing_point": -2,
+    "fun": -50
   }
```

Comment:
The `liquid_soap` seemed to me the simplest way of adding things like, shampoo, shower gel,  hair conditioner and hand-wash without having a new liquid for each one. I am just not sure how to spawn it in to a new container for the hand wash.  Any suggestions on this?